### ISSUE TITLE
Sample war doesn't load

### ISSUE THREAD

**Issue by bdw429s** [cfmlprojects/runwar#14]
This sample war provided by Tomcat won't load in CommandBox/runwar.  Seems we make some assumptions about the existence of jars in the lib folder?

https://tomcat.apache.org/tomcat-7.0-doc/appdev/sample/sample.war
```
CommandBox:Downloads> start WARPath=sample.war
C:\Users\Brad\Downloads\sample.war
The server for C:\Users\Brad\Downloads is starting on 127.0.0.1:3549... type 'server status' to see result
CommandBox:Downloads> server log
******************************************************************************
Starting - port:3549 stop-port:3550 warpath:C:\Users\Brad\Downloads\sample
context: /  -  version: 3.4.4
web-dirs: C:\Users\Brad\Downloads\sample
Log Directory: C:\Users\Brad\.CommandBox\server\2C8E41FA2D1D800F608DCC42D65E53A5-Downloads\logs
******************************************************************************
Exception in thread "main" java.lang.RuntimeException: FATAL: Could not load any libs for war: C:\Users\Brad\Downloads\sample
        at runwar.Server.startServer(Server.java:399)
        at runwar.Start.main(Start.java:12)
```

**Comment by bdw429s** [comment]
Same  issue with https://glassfish.dev.java.net/downloads/quickstart/hello.war

**Comment by denuno** [comment]
Should be fixed in 3.4.5, building shortly here.
